cablelabs/lpwanserver
# Issue 40

#### What does this PR do?

Progress toward Issue #40
* Support networkProtocol specific dynamic field input for network create
* added eslint

#### Do you have any concerns with this PR?

I will likely evolve to a different .eslintrc

#### How can the reviewer verify this PR?

* Visual inspection
* Manual test if you want
   - pull and build the code, and see if you can create a LoraOS network, and a company 

#### Any background context you want to provide?

n/a

#### Screenshots or logs (if appropriate)

n/a

#### Questions:

- Have you connected this PR to the issue it resolves?
yes

- Does the documentation need an update?
no

- Does this add new dependencies?
yes: ramda (js util) and eslint packages

- Have you added unit or functional tests for this PR?
no
